backend/dal/Services/UserService.cs

```diff
@@ -81,7 +81,7 @@ public PimsUser Activate()
                 this.Logger.LogInformation($"User Activation: key:{key}, email:{email}, username:{username}, first:{givenName}, surname:{surname}");
 
                 var person = new PimsPerson() { Surname = surname, FirstName = givenName };
-                this.Context.PimsPeople.Add(person);
+                this.Context.PimsPerson.Add(person);
                 this.Context.CommitTransaction();
 
                 user = new PimsUser()
```

Comment:
To support my previous comment - this line is correct (in my opinion). Every user that we activate should be added to the PimsUser table (note this is different from PimsPerson)